src/store/modules/currentFarmer/state.ts

```diff
@@ -0,0 +1,7 @@
+import { CurrentFarmerState } from './types';
+
+const initialState: CurrentFarmerState = {
+  currentFarmerUUID: 'tempFrom345',
```

Comment:
For this I would just leave the initial state as empty or null or however you setup the reducer to deal with no active farmer

Reply:
Oh yeah this was from me testing it I forgot to delete it